src/app/public/modules/progress-indicator/fixtures/progress-indicator.component.fixture.html

```diff
@@ -28,6 +28,12 @@
     Waterfall content 3
   </sky-progress-indicator-item>
 
+  <sky-progress-indicator-item *ngIf="showFourthItem"
+    title="Do the forth thing"
```

Comment:
Sorry, I know this is a test, but I just can't...

Spelling > `forth`

Reply:
Fixed